app/import/robot/schema/map.js

```diff
@@ -0,0 +1,35 @@
+const map = {
+  Owner: {
+    owner: {
+      FirstName: 'firstName',
+      LastName: 'lastName',
+      DateOfBirth: 'birthDate',
+      Address: {
+        address: {
```

Comment:
Why the nested 'address'? Was the outer meant to be an array called 'Addresses'?

Reply:
It's due to how the maps work in read-excel-file. Outer key is the pseudo excel element and the inner key is the key for the generated object.